HostOcean/HostOcean.Api/Startup.cs

```diff
@@ -21,14 +21,26 @@ public Startup(IConfiguration configuration)
         }
 
         public IConfiguration Configuration { get; }
-        
+
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+
+            var connection = Configuration.GetConnectionString("DefaultConnection");
+            services.AddDbContext<HostOceanDbContext>(options => options.UseSqlServer(connection, b => b.MigrationsAssembly("HostOcean.Persistence")));
+
+            services.AddIdentity<User, IdentityRole>()
+                .AddEntityFrameworkStores<HostOceanDbContext>()
+                .AddDefaultTokenProviders();
+
+            services.AddScoped(typeof(IRepository<>), typeof(Repository<>));
+            services.AddScoped<IUnitOfWork, UnitOfWork>();
         }
-        
+
         public void Configure(IApplicationBuilder app, IHostingEnvironment env)
         {
+            HostOceanDbInitializer.Initialize(app.ApplicationServices.GetRequiredService<IServiceScopeFactory>().CreateScope().ServiceProvider);
```

Comment:
Is it possible to seed database by google calendar event by injection BsurGroupSevice implemented in #17?